azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/AbstractAzureMojo.java

```diff
@@ -274,6 +282,20 @@ public Azure getAzureClient() throws AzureAuthFailureException {
         return azure;
     }
 
+    protected AuthType getAuthTypeEnum() {
+        if (StringUtils.isEmpty(authType)) {
+            return AuthType.EMPTY;
+        }
+        AuthType result = Arrays.stream(AuthType.values())
+                .filter(authTypeEnum -> StringUtils.equalsAnyIgnoreCase(authTypeEnum.name(), authType))
+                .findFirst().orElse(null);
+        if (result == null) {
+            Log.warn(String.format(INVALID_AUTH_TYPE, authType));
```

Comment:
Please list all available values for auth type since here user may encounter difficulties for auth-type.

Reply:
Done.